inheritance/lib/src/test/java/inheritance/LibraryTest.java

```diff
@@ -7,13 +7,17 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 class LibraryTest {
-    @Test void restaurantClassConstructorWorks() {
```

Comment:
would recommend using an 

```Java
@BeforeEach
  void setUp() {
    // create the list of restaurants here then it will do all tests on this list.
  }
```
in your tests if they should be performing on a restaurant so you can see how it reflects over time.